roles/logs/tasks/main.yml

```diff
@@ -45,3 +45,10 @@
 - name: copy APT User-Agent configuration
   copy: src=apt-user-agent.conf dest=/etc/apt/apt.conf.d/99useragent mode=755
 
+- name: Ensure Anacron will run on battery
+  lineinfile: 
+    dest: /etc/default/anacron
+    regexp: '^ANACRON_RUN_ON_BATTERY_POWER'
+    line: 'ANACRON_RUN_ON_BATTERY_POWER=yes'
```

Comment:
Why not, I'm not sure anacron will be able to read the battery specific path `/sys/power/axp_pmu/battery/*`

Reply:
Good point, unfortunately I was not able to test this. Maybe I should just remove this.